internal/wrappers/results-http.go

```diff
@@ -17,8 +17,8 @@ const (
 	sort                    = "sort"
 	sortResultsDefault      = "-severity"
 	offset                  = "offset"
-	astAPIPageLen           = 1000
-	astAPIPagingValue       = "1000"
+	astAPIPageLen           = 10_000
```

Comment:
what is the meaning of this value?10_000

Reply:
syntactic sugar for 10000

Reply:
for better readbility

Reply:
great
so add _ also in the next line astAPIPagingValue = "10000"